packages/auth/gulpfile.js

```diff
@@ -28,10 +28,18 @@ const File = require('vinyl');
 const OPTIMIZATION_LEVEL = 'ADVANCED_OPTIMIZATIONS';
 
 // For minified builds, wrap the output so we avoid leaking global variables.
-const CJS_WRAPPER_PREFIX = `(function() {var firebase = require('@firebase/app').default;`;
+const CJS_WRAPPER_PREFIX =
+  `(function() {var firebase = require('@firebase/app').default;`;
 const EMS_WRAPPER_PREFIX = `import firebase from '@firebase/app';(function() {`;
-const WRAPPER_SUFFIX = `}).call(typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : typeof window !== 'undefined' ? window : {});`;
+const WRAPPER_SUFFIX =
+  `}).call(typeof global !== 'undefined' ? ` +
+  `global : typeof self !== 'undefined' ? ` +
+  `self : typeof window !== 'undefined' ? window : {});`;
```

Comment:
Same here for above block:
Add 2 more spaces here. (if right hand side of = is on second line)
```
const WRAPPER_SUFFIX =
    'value1' +
    'value2';
```